controllers/users.js

```diff
@@ -2,7 +2,8 @@ const User = require("../models/user");
 const { getPixelatedImage } = require("../middleware");
 const { ROUTES } = require("./routes");
 
-const extraStyles = '<link rel="stylesheet" href="/stylesheets/cards.css" />';
+const style = require("../public/javascripts/extraStyles.js");
+const extraStyles = style.cardStyle;
```

Comment:
You can destructure the key variable pair from the import above so no need to declare this constant:

```const { cardStyle } = require("../public/javascripts/extraStyles.js");```

Reply:
is it better to destructure in cases where more than one extra style is being used? (main.js)

Reply:
Yeah, generally it's better, you can descructure whatever you need. `const { cardStyle, mainStyle } = require('../public/javascripts/extraStyles.js"`. 

Can also just import the `styles` object and use it like this: `extraStyles: styles.cardStyle` etc.